getting-started-2.0.md

```diff
@@ -0,0 +1,160 @@
+
+
+# Beginner's Tutorial
+This guide helps you get started with Helix with very little effort, then progressively discover more features as you need them.
+
+
+## Step 1: Publish your content (look ma, no hands!)
+
+> You'll need:
+> - a [GitHub](https://github.com/join) account
+> - basic [Markdown](https://guides.github.com/features/mastering-markdown/) knowledge
+
+The easiest way to publish content is to have [_Helix Pages_](https://www.project-helix.page/) render and deliver a Markdown file in your GitHub repository:
+
+1. Click the _Create new repository_ button to set up a new public [GitHub repository](https://github.com/new/).  
+![](./dummy.png "TODO: Create a repository")
+1. Click the _Create new file_ button in your GitHub repository to add a new file called `index.md`. This will be your homepage.  
+![](./dummy.png "TODO: Create an index.md")
+1. Add some great Markdown content. Feel free to use [this example](https://raw.githubusercontent.com/adobe/helix-example-first/master/index.md "example index.md") for starters.
+1. Click the _Commit changes_ button.
+1. Go to `https://<my-repo>-<my-name>.project-helix.page/` to admire your work. Replace `<my-repo>` and `<my-name>` with your actual repository and username:  
+![](./dummy.png "TODO: Helix Pages rendering your content")
+1. **Congratulations!** 🎉 You have just published your first page using _Helix Pages_!
+
+> **Hint:** You can add as many Markdown files as you like, and link them in Markdown.
+
+> **Note:** If your GitHub username contains one or more dashes, add an extra dash between the repository and username in the _Helix Pages_ URL. For example, if your user is called `meet-me`, and your repository is `test`, the corresponding _Helix Pages_ URL would be `https://test--meet-me.project-helix.page/`.
+
+## Step 2: Customize header and footer
+
+_Helix Pages_ can render a header and footer based on Markdown content in your GitHub repository:
+
+1. Add a file called `header.md` to your GitHub repository.  
+![](./dummy.png "TODO: Create a header.md")
+1. Add a logo, and maybe a list of other links you would like to show in the header navigation. Here's some [inspiration](https://raw.githubusercontent.com/adobe/helix-example-first/master/header.md "example header.md").
+1. Click the _Commit changes_ button.
+1. If you added a logo in step 2, upload that now.  
+![](./dummy.png "TODO: Upload logo")
+1. Add a file called `footer.md` to your GitHub repository.
+1. Add some content you would like to be shown in the footer. Here's a [proposal](https://raw.githubusercontent.com/adobe/helix-example-first/master/footer.md "example footer.md").
+1. Click the _Commit changes_ button.
+1. Reload `https://<my-repo>-<my-name>.project-helix.page/` to see header and footer in action.  
+![TODO: Helix Pages rendering your header and footer](./dummy.png)
+
```

Comment:
I would insert a new step 3: Use a branch

```suggestion
## Step 3: Use a branch

By default, _Helix Pages_ renders the `master` branch of your GitHub repository. It can also render any branch of your repository using the pattern `https://<my-branch>--<my-repo>--<my-name>.project-helix.page/` (note the two dashes, they are required).

1. Create a branch.
![TODO: Create a branch.md](./dummy.png)
1. Edit the `index.md` file in the new branch.
![TODO: Edit index.md in the branch](./dummy.png)
1. Click the _Commit changes_ button.
1. Load https://<my-branch>--<my-repo>--<my-name>.project-helix.page/
![TODO: Helix Pages rendering your branch](./dummy.png)
```

Reply:
Naturally, this shifts all the numbers in the titles :)

Reply:
Good idea! But how about introducing it at step 2, where you still only focus on `index.md` and now get to play with variants?

Reply:
That was my initial thought but I realised that this is a more "advanced" concept. header/footer are probably more of interest for a beginner.

Reply:
I just played around a bit with separate step, substep of step 1, and making it a hint at the end of step 1, and I think I prefer the latter: it keeps the tutorial flow intact without introducing too much complexity and extra steps with screenshots etc, and contains a link to details about branching for those who are interested.

Let me know what you think!